CRM/Report/Form/Contribute/Bookkeeping.php

```diff
@@ -536,8 +537,10 @@ public function where() {
             $relative = $this->_params["{$fieldName}_relative"] ?? NULL;
```

Comment:
@sunilpawar everything above this line looks fine. But on the where clause - could we just stop overriding it?

We'd need to add an implementation of whereClause() to add the handling of 
     'credit_accounting_code',
      'credit_name',
      'credit_contact_id',

Reply:
@eileenmcnaughton added these field in `whereClause` clause .

Existing functionality does work by giving date range with time but it always from   `20200827000000` to `20200827235959` , there was no control on time field to specify it.

Reply:
@sunilpawar thanks for confirming - your final fix will fix more than just this report I think